actix-web/postgres/Cargo.toml

```diff
@@ -8,6 +8,6 @@ actix-web = "4.3.1"
 shuttle-actix-web = "0.36.0"
 shuttle-runtime = "0.36.0"
 serde = "1.0.148"
-shuttle-shared-db = { version = "0.36.0", features = ["postgres"] }
-sqlx = { version = "0.7.1", features = ["runtime-tokio-native-tls", "postgres"] }
```

Comment:
It doesn't harm to keep these feature flags on `sqlx` right? I'm thinking a normal non-shuttle app would have them so having them just makes it clearer that we don't have a vendor lock-in.

Reply:
Switching shared db from `postgres` to `postgres-rustls` would then activate both tls features on sqlx while only one is being used. I see it as shared-db steering which features are activated, and any extras can be activated out here.